data-generation/src/main/scala/utils/UtilsUtils.scala

```diff
@@ -14,4 +15,48 @@ object UtilsUtils {
   def getConfig(key: String): String = sys.env(key)
 
   def getConfigSafe(key: String): Option[String] = sys.env.get(key)
+
+  def loadThreshold(): Int = UtilsUtils.getConfigSafe("THRESHOLD") match {
+    case Some(value) => value.toInt
+    case None => 40
+  }
+
+  /**
+   * Try to load data source. Return ContactsHyperText as default.
+   */
+  def loadDataSource(): DataSource = UtilsUtils.getConfigSafe("DATA_SOURCE") match {
+    case Some(value) =>
+      if (value.toInt == 1) return DataSource.Reptilian
+      if (value.toInt == 1) return DataSource.ContactsHyperText
+      if (value.toInt == 1) return DataSource.FbMessages
+      DataSource.Reptilian
+    case None => DataSource.ContactsHyperText
+  }
+
+  def loadTimestamp(): Long = UtilsUtils.getConfigSafe("TIMESTAMP") match {
+    case Some(value) => value.toLong
+    case None => 0
+  }
```

Comment:
```suggestion
  def loadTimestamp(): Long = UtilsUtils.getConfigSafe("TIMESTAMP").getOrElse("0").toLong
  def loadTimestamp(): Long = UtilsUtils.getConfigSafe("TIMESTAMP").map(toLong).getOrElse(0L)
```
Dette er også to muligeter. Det som er nice med Options er at de støtter .map
(Men dette er kun en kommentar på syntax, så er ikke viktig)